### ISSUE TITLE
[UX] Add Member page need to have "search" user input

### ISSUE THREAD

**Issue by RomanGalochkin** [foretold-app/foretold#667]
The simplest form of an input for searching members.
![Screenshot_20190913_102534](https://user-images.githubusercontent.com/16107421/64844559-d30a5000-d610-11e9-8311-9672dfc1b5cd.png)

**Comment by RomanGalochkin** [comment]
If there are some way of adding new searching filed right into "PaginationFunctor" it would be good.

**Comment by OAGr** [comment]
Do you have ideas on how we could do search with the backend? Have graphQL - PostgreSQL do a fuzzy search on name? 

Search is definitely a thing to be done.

Another question is that maybe we should use Algolia or ElasticSearch or similar, and push to do that now, then do it for all search.

**Comment by RomanGalochkin** [comment]
I do not see here huge issues. All we need is to send new argument for "getAll" or "getConnection" method and search within specified field.

No we do not need ES or similar here. It does not have a sense.

**Comment by OAGr** [comment]
Hm... that seems pretty good. I think this is a bit lower on the priority list then some other semi-obvious stuff; for instance, having a search bar in the measurables page, but will keep it in mind.

**Comment by RomanGalochkin** [comment]
Yes sure, I just wanted to note this because do not like to rely on my memory.

**Comment by OAGr** [comment]
Makes sense
